pom.xml

```diff
@@ -524,6 +525,9 @@
                             <exclude>software.amazon.awssdk.thirdparty.*</exclude>
                             <exclude>software.amazon.awssdk.services.s3.checksums.ChecksumCalculatingAsyncRequestBody</exclude>
                             <exclude>software.amazon.awssdk.protocols.core.OperationInfo</exclude>
+                            <exclude>software.amazon.awssdk.regions.ServiceMetadata</exclude>
+                            <exclude>software.amazon.awssdk.regions.servicemetadata.*ServiceMetadata</exclude>
+                            <exclude>software.amazon.awssdk.regions.DefaultServiceMetadata</exclude>
```

Comment:
Yes

Reply:
It's being too picky about backwards-compatible. There weren't any actual breaking changes in these classes, since we added the implementation in the parent interface.